coolbeans/coolbeansapp/models.py

```diff
@@ -39,9 +39,8 @@ class Order(models.Model):
     total_price = models.FloatField(null=True)
     date = models.DateTimeField(auto_now_add=True)
     status = models.CharField(max_length=200, null=True, choices=STATUS)
-    is_confirmed = models.BooleanField(default=False)
```

Comment:
Mel - I had added line 42 because i was just thought-experimenting and i forgot to take it out.  If you decided to use it in your code, i can undo the deletion but i wasn't trying to impose my way on you. You can easily use the status options to achieve the same thing.